data_loaders/cifar_data_loader.py

```diff
@@ -0,0 +1,93 @@
+import os
+import pickle
+from cfg_loader import ConfigLoader
+import numpy as np
+import polars as pl
+import torch
+from base_data_loader import BaseDataLoader
+
+
+class CifarDataLoader(BaseDataLoader):
```

Comment:
CifarDataLoader name indicates that it works for both CIFAR 10 and 100. But load_data will not work for CIFAR 100